scripts/comp_sql.c

```diff
@@ -107,6 +107,18 @@ static void print_query(FILE *out, const char *query)
       column++;
       break;
     default:
+      /* Fix clang-8 warning -Winvalid-source-encoding.
```

Comment:
Please, report this to both Percona Jira and upstream as well.

Reply:
Reported at:
https://jira.percona.com/browse/PS-5836
https://bugs.mysql.com/bug.php?id=96449